docs/source/specification.md

```diff
@@ -100,6 +100,9 @@ project. The *Broad* datatype for that category *must* no longer be used.
 If you have a modality that does not fit into the current datatype options,
 please get in contact!
 
+**behav**
+- `motion`: motion capture
```

Comment:
I 'd rather use more general term "motion tracking", and then give some concrete examples to clarify the scope.

```suggestion}
- `motion`: motion tracking, e.g. from marker-based motion capture systems, marker-less pose estimation, or inertial measurement units (IMUs)
```

Quote from the [BIDS-Motion paper](https://www.nature.com/articles/s41597-024-03559-8):
> Motion-BIDS is compatible with a wide range of motion tracking systems. Therefore, the term “motion tracking” in this context is used in a broad sense, encompassing the recording of the movement of an object in physical space, commonly referred to as “motion capture” or “pose estimation”, as well as in virtual space.